### ISSUE TITLE
Array union instead of array_merge in menu.inc (the navbar part)

### ISSUE THREAD

**Issue by zanami** [drupalprojects/kalatheme#211]
Array + array here does nothing if the first array isn't empty (and it's not actually)

$link['#options']['attributes']['class'] += array("dropdown-toggle");

[menu.inc](https://github.com/drupalprojects/kalatheme/blob/767da8045bc9a7691e3b80333306f935fbbcd1ba/includes/menu.inc#L126)

> The + operator returns the right-hand array appended to the left-hand array; for keys that exist in both arrays, the elements from the left-hand array will be used, and the matching elements from the right-hand array will be ignored.

[PHP Array Operators](http://php.net/manual/en/language.operators.array.php)

Some other places with Array += Array (but I don't know if there's a problem there)

> includes/libraries.inc:  $options += array(
> includes/fapi.inc:  $element += array(
> includes/core.inc:  $uri['options'] += array(

**Comment by soniktrooth** [comment]
@zanami I agree with the logic here but are you seeing a specific instance where `$link['#options']['attributes']['class']` is empty? Is it throwing an error or a break in functionality that you can describe?
